src/Servers/Kestrel/tools/CodeGenerator/HttpProtocolFeatureCollection.cs

```diff
@@ -67,7 +59,7 @@ public static string GenerateFile()
             {
                 "IHttpRequestFeature",
                 "IHttpResponseFeature",
-                "IResponseBodyPipeFeature",
+                "IHttpResponseBodyFeature",
                 "IRequestBodyPipeFeature",
```

Comment:
Remove?

Reply:
I'm not doing the request in this PR, only the response. The request should be much simpler.

Reply:
OK